mypy/modulefinder.py

```diff
@@ -49,10 +49,18 @@ class ModuleNotFoundReason(Enum):
     # corresponding *-stubs package.
     FOUND_WITHOUT_TYPE_HINTS = 1
 
+    # The module was not found in the current working directory, but
+    # was able to be found in the parent directory.
+    WRONG_WORKING_DIRECTORY = 2
+
     def error_message_templates(self) -> Tuple[str, str]:
         if self is ModuleNotFoundReason.NOT_FOUND:
             msg = "Cannot find implementation or library stub for module named '{}'"
             note = "See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports"
+        elif self is ModuleNotFoundReason.WRONG_WORKING_DIRECTORY:
+            msg = "Cannot find implementation or library stub for module named '{}'"
+            note = ("You may be running mypy in the wrong directory, "
```

Comment:
Some brainstorm: `You may be running mypy in a subdirectory, mypy should be run in the package root directory`. @JukkaL may have better suggestions.

Reply:
Any suggestions on this message, or other changes that need to be made in order for this PR to be ready for merge @JukkaL?